### ISSUE TITLE
Add keybind menu

### ISSUE THREAD

**Issue by j-huff** [Pattern-Projector/pattern-projector#161]
With with the addition of current and future keybinds (#130 #154 ), it would be good to add some type of menu that will list the available keybinds. We could also add the ability to configure the keybinds with this menu + a reset to default settings button.

I'm unsure exactly how this menu should be opened. If we plan on ever adding a more general settings menu, then this would just be a tab/page in that menu.

If we don't feel like going that far, then just a little togglable static popup showing the (non-configurable) keybinds and maybe some other useful tips could be good.

**Comment by courtneypattison** [comment]
I agree that we need to show people what the keybinds are! I’m not sure what the best method is. I’m trying to think of when I learn to use them and most of the time I only start using keybinds after seeing them in a drop-down menu. Would it make sense to add the keybinds in parens in the tooltips? Or something like that?

**Comment by j-huff** [comment]
> Would it make sense to add the keybinds in parens in the tooltips?

Yes I like that. However there are still some keybinds that wouldn't be covered by this, such as the arrow keys (seems self explanatory but you never know) and the ones I added in #154.

**Comment by j-huff** [comment]
For me, I know that I would likely never read the landing page lol.

> Maybe a drop-down from the info icon?

Possibly switching the "info icon" to a "three dot" icon, and clicking it opens a dropdown menu. This could be a more visually compressed version of the dropdown menu used by the overlay button, instead just showing icons (each with a tooltip) instead of icons + text. So you would have a menu item for going to the landing page and another one to open the keybind menu.

**Comment by hluedeke** [comment]
I think most of the keybinds should go in the tooltips. The other ones are probably fine "in the docs" under a Reference/Keybinds section (or similar).

**Comment by courtneypattison** [comment]
> I think most of the keybinds should go in the tooltips. The other ones are probably fine "in the docs" under a Reference/Keybinds section (or similar).

I like this option because it's simple! We can add more menu items when we really need them. That info page needs some love so I'll try to add this section when I work on that.

**Comment by courtneypattison** [comment]
Added mention of current hotkeys to landing page in #224
